.github/workflows/vuln.yml

```diff
@@ -2,12 +2,15 @@ name: Vulnerability Check
 
 on: [push, pull_request]
 
+env:
+  GO_VERSIONS: ${{ steps.get_go_version.outputs.versions }}
+
 jobs:
   test:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        go: ['1.19.6', '1.20.1']
+        go: ${{ env.GO_VERSIONS }}
```

Comment:
1. GO_VERSIONS is determined after `get_go_version.sh` is executed, right?
2. GO_VERSIONS will be like `go1.19.6`, not `1.19.6`
3. This doesn't work when the version is for example `1.21`. actions/setup-go@v3 requires `1.21.0` in this case.

Reply:
1. yes!
2. changing it right now. will do a commit soon.